server/tests/db/test_01_es6.js

```diff
@@ -327,6 +338,132 @@ tests.NumberMaxSafeInteger = function() {
       'Number.isSafeInteger(Number.MAX_SAFE_INTEGER + 1);');
 };
 
+tests.NumberMinSafeInteger = function() {
+  console.assert(Number.MIN_SAFE_INTEGER - 1 === -Math.pow(2, 53),
+      'Number.MIN_SAFE_INTEGER');
+  console.assert(Number.isSafeInteger(Number.MIN_SAFE_INTEGER),
+      'Number.isSafeInteger(Number.MIN_SAFE_INTEGER);');
+  console.assert(!Number.isSafeInteger(Number.MIN_SAFE_INTEGER - 1),
+      'Number.isSafeInteger(Number.MIN_SAFE_INTEGER - 1);');
```

Comment:
(But this test is O.K., because a "safe" integer can't be the result of rounding!)